### PR TITLE
QUICK-FIX: Fix issues after the merge, clean up

### PR DESCRIPTION
**Pull request by samotarnik**

### CODE REVIEW

File: src/ggrc/models/request.py [hunk 1]
```diff
@@ -65,8 +65,8 @@ class Request(AutoStatusChangable, Assignable, Documentable, Personable,
   # TODO Make status via Stateful Mixin
   status = deferred(db.Column(db.Enum(*VALID_STATES), nullable=False,
                               default=tuple(START_STATE)[0]), 'Request')
-  requested_on = deferred(db.Column(db.Date, nullable=False), 'Request')
-  due_on = deferred(db.Column(db.Date, nullable=False), 'Request')
+  start_date = deferred(db.Column(db.Date, nullable=False), 'Request')
```
Comment (edofic) [inline src/ggrc/models/request.py:68]: @samotarnik didn't you already perform this change on the `release/candidate` branch? How did it turn back? Automerge maybe?

**Comment by edofic** [comment]
Ran migrations, unit & integration tests. Clicked around for a bit.
looks good to me :+1:  (but I'd still like an explanation)

File: src/ggrc/models/request.py (hunk 1 above)
Comment (samotarnik) [inline src/ggrc/models/request.py:68]: i think so, yeah... something's a bit weird, went through commits in `gitk` and one commit has the new code, while the next commit has the obsolete attrs again without any changes to file shown =/

**Comment by anze3db** [comment]
It was most likely due to merge conflicts, I'm merging here, you can continue the discussion on the ggrc-core repo.

**Pull request merged**